### PR TITLE
Update docker publishing to match documentation

### PR DESCRIPTION
**Pull request by jsuereth**

### CODE REVIEW

**Review by lquerel — APPROVED** [review]

File: .github/dependabot.yml [hunk 1]
```diff
@@ -1,6 +1,12 @@
...
+  # Update cargo packages
   - package-ecosystem: "cargo" 
     directory: "/"
     schedule:
       interval: "daily"
+  # Update github action values
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "daily"
```
Comment (sfc-gh-bdrutu) [inline .github/dependabot.yml:12]: New line.

File: .github/workflows/publish-docker.yml [hunk 2]
```diff
@@ -24,31 +24,23 @@ jobs:
     runs-on: ubuntu-latest
     needs: tests
     steps:
-    - uses: actions/checkout@v4
-    - name: Build the Docker image
-      run: docker build . -t weaver
-    - name: Push the Docker image
-      if: github.ref == 'refs/heads/main' || startsWith(github.ref, 'refs/tags/')
```
Comment (sfc-gh-bdrutu) [inline .github/workflows/publish-docker.yml:31]: You remove the condition to run this only when merged to main or when tags are added, is this intended?

Comment (jsuereth) [reply .github/workflows/publish-docker.yml:31]: for debugging only.  The login isn't working, but I figured out why the previous solution wasn't working.

I'm not sure why the SAME secrets won't work with the login action.

**Comment by jsuereth** [comment]
Closing this for now - will pick it up later, when we want to improve docker publishing.

**Pull request closed without merging**